api/kroxylicious-filter-api-bom/.gitignore

```diff
@@ -0,0 +1,38 @@
+target/
```

Comment:
I've no experience with writing BOMs, so excuse the dumb questions.  Why does the module need its own .gitignore?

Reply:
Nothing to do with the bom, was Auto generated by IntelliJ when creating the module. 

Happy to remove it.